fe1-web/src/features/rollCall/screens/RollCallScanner.tsx

```diff
@@ -30,14 +30,28 @@ import { makeRollCallSelector } from '../reducer';
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    alignItems: 'center',
     justifyContent: 'space-between',
     marginVertical: Spacing.contentSpacing,
   } as ViewStyle,
   qrCode: {
+    alignItems: 'center',
     opacity: 0.5,
   } as ViewStyle,
-  enterManually: {} as ViewStyle,
+  counter: {
+    alignItems: 'flex-start',
```

Comment:
Is it really needed since the `alignItems: 'center'` on the container has been removed ? It seems to me that the StyleSheets could be simplified.

Reply:
The answer is linked to my comment above, I couldn't find a way to simplify it

Reply:
Ok, this is good then.